src/mon/Elector.cc

```diff
@@ -492,7 +492,7 @@ void Elector::ping_check(int peer)
 {
   dout(20) << __func__ << " to peer " << peer << dendl;
 
-  if (peer >= mon->monmap->ranks.size()) {
+  if (peer >= std::ssize(mon->monmap->ranks)) {
```

Comment:
or just
```c++
if (peer >= ssize(mon->monmap->ranks)) {
```

as ADL can pick `std::ssize()` for us. because `ranks` is a `std::vector<>`.